go.sum

```diff
@@ -5,23 +5,32 @@ github.com/golang/mock v1.2.0 h1:28o5sBqPkBsMGnC6b4MvE2TzSr5/AT4c/1fLqVGIwlk=
 github.com/golang/mock v1.2.0/go.mod h1:oTYuIxOrZwtPieC+H1uAHpcLFnEyAGVDL/k47Jfbm0A=
 github.com/golang/protobuf v1.2.0 h1:P3YflyNX/ehuJFLhxviNdFxQPkGK5cDcApsge1SqnvM=
 github.com/golang/protobuf v1.2.0/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
-github.com/google/go-github v3.0.0+incompatible h1:lsf+ZqlC8Jm53WVfjFavvCCIamHR13YTWhhkB9j549Y=
-github.com/google/go-github v3.0.0+incompatible/go.mod h1:zLgOLi98H3fifZn+44m+umXrS52loVEgC2AApnigrVQ=
+github.com/google/go-github v17.0.0+incompatible h1:N0LgJ1j65A7kfXrZnUDaYCs/Sf4rEjNlfyDHW9dolSY=
```

Comment:
Please run `go mod tidy` which should delete the v17 lines and anything else that's no longer needed.

Reply:
`go mod tidy` does not remove this line, even if I delete the `go.sum` file and re-generate it, this line is included again, and if I remove it manually any `go mod` related command like `go get` will also include it again.

Reply:
Interesting, I guess it's a transitive dependency for one of our dependencies or something else. Thanks for looking into it!